openedx/core/djangoapps/content_tagging/rest_api/v1/serializers.py

```diff
@@ -99,3 +99,10 @@ class Meta:
         model = TaxonomySerializer.Meta.model
         fields = TaxonomySerializer.Meta.fields + ["orgs", "all_orgs"]
         read_only_fields = ["orgs", "all_orgs"]
+
+
+class ContentObjectChildrenTagsExportQueryParamsSerializer(serializers.Serializer):  # pylint: disable=abstract-method
```

Comment:
This is starting to look like a Java class name :) Can it be a little shorter?

```suggestion
class ExportContentTagsQueryParamsSerializer(serializers.Serializer):  # pylint: disable=abstract-method
```

Reply:
I had the exact same feeling @pomegranited! :rofl: 
Thank you for helping with a more sane name!
Fix: 8f1238e56605ce3b19402b2b5c692ac472bc1f57